File: libraryWallrCustoms/src/main/java/com/zebrostudio/wallrcustoms/customtextview/FontCache.java
package com.zebrostudio.wallrcustoms.customtextview;

import android.content.Context;
import android.graphics.Typeface;

class FontCache {

  static Typeface getRobotoRegularTypeface(Context context) {
    return Typeface.createFromAsset(context.getAssets(), "fonts/Roboto-Regular.ttf");
  }
}

File: libraryWallrCustoms/src/main/java/com/zebrostudio/wallrcustoms/customtextview/FontCache.kt
package com.zebrostudio.wallrcustoms.customtextview

import android.content.Context
import android.graphics.Typeface

internal object FontCache {

  fun getRobotoRegularTypeface(context: Context): Typeface {
    return Typeface.createFromAsset(context.assets, "fonts/Roboto-Regular.ttf")
  }

}